pkgs/tools/X11/smart-wallpaper/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, makeWrapper
+, xdpyinfo
+, killall
+, xwinwrap
+}:
+
+stdenv.mkDerivation {
+  pname = "smart-wallpaper";
+  version = "unstable-2022-09-01";
+
+  src = fetchFromGitHub {
+    owner = "Baitinq";
+    repo = "smart-wallpaper";
+    rev = "d175695d3485fb14144c1908eb3569b20caa6ba5";
+    sha256 = "sha256-cFgvuntdKPzdQJ7xE2DIT+aNAazocIhM6BBbcQOlryU=";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp smart-wallpaper $out/bin/smart-wallpaper
+    wrapProgram $out/bin/smart-wallpaper \
+      --prefix PATH : ${lib.makeBinPath [ xdpyinfo killall xwinwrap ]}
+  '';
+
+  meta = with lib; {
```

Comment:
Given that this uses X11 tools, are you sure that this can run on MacOS?

Reply:
I dont think it can, Ill add it as a linux platform then:) Thank you.